build.gradle

```diff
@@ -5,7 +5,7 @@ buildscript {
         mavenCentral()  //  This is the Maven Central repo
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:1.3.0'
+        classpath 'com.android.tools.build:gradle:1.5.0'
```

Comment:
Looks like this upgrade to the Gradle plugin causes the build to fail due to Retrofit being included both in libs, and linked from Maven Central. I'm compiling a pull request cleaning up and upgrading our libraries that should fix this.